### PR TITLE
vgsvg: html escape text

### PR DESCRIPTION
**Pull request by chmike**
Please take a look.

PR for issue #511

### CODE REVIEW

**Comment by sbinet** [comment]
what about the other string arguments ? (in `DrawImage`, in `style`, in elmt`)

**Comment by sbinet** [comment]
also: please modify the commit message to read `vg/vgsvg: ...`
(and add `Fixes gonum/plot#511.` in the body of the commit message)

**Comment by chmike** [comment]
@binet 
- `elm()` use predefined strings. No user provided text is involved.
- `style()` dito.
- `DrawImage()` gets an image object. The image is base64 encoded. No user provided text is involved.

**Comment by kortschak** [comment]
Can we please have a test for this?

**Comment by chmike** [comment]
@kortschak can we first fix the issue #506 that I had to reopen ? When trying to view the generated SVG on CodePen I got an invalid font size. When specifying the font size unit as *px*, the problem is solved in CodePen.

**Comment by chmike** [comment]
@kortschak @sbinet ping ?

File: vg/vgsvg/vgsvg.go [hunk 1]
```diff
@@ -290,6 +291,8 @@ func large(a float64) int {
 	return 0
 }
 
+// FillString draws str at position pt using the specified font.
+// str is escaped with html.EscapeString.
```
Comment (kortschak) [inline vg/vgsvg/vgsvg.go]: I'd prefer the wording I suggested for stylistic reasons.

**Comment by binet** [comment]
Please stop sending me messages, I'm not part of this conversation at all.
I don't have the slightest idea of what you're talking about...

________________________________
De : Christophe Meessen <notifications@github.com>
Envoyé : vendredi 25 janvier 2019 11:39
À : gonum/plot
Cc : binet; Mention
Objet : Re: [gonum/plot] vgsvg: html escape text (#512)


@kortschak<https://github.com/kortschak> @sbinet<https://github.com/sbinet> ping ?

**Comment by sbinet** [comment]
right. I am `sbinet`, not `binet`.

**Comment by sbinet** [comment]
@binet you can remove yourself from this conversation by going on this web page and click `Unsubscribe` (top-right)

File: vg/vgsvg/vgsvg.go (hunk 1 above)
Comment (chmike) [reply vg/vgsvg/vgsvg.go]: The function comment convention is to start the comment by the function name: `FillString ...`. 

Would we converge if I put this ?
``` Go
// FillString escape passed text with html.EscapeString.
```

Comment (chmike) [reply vg/vgsvg/vgsvg.go]: See here : https://blog.golang.org/godoc-documenting-go-code

Comment (kortschak) [reply vg/vgsvg/vgsvg.go]: See https://github.com/gonum/plot/pull/512#pullrequestreview-195727529

```
// FillString draws str at position pt using the specified font.
// Text passed to FillString is escaped with html.EscapeString.
```

**Comment by kortschak** [comment]
@chmike Pushes to forked repos don't result in notifications. If you want to guarantee that someone will look after an amendment, please post a "PTAL".

**Comment by kortschak** [comment]
Thanks

**Pull request merged**